package-lock.json

```diff
@@ -1,7722 +1,8 @@
 {
     "name": "dynamodb-onetable",
     "version": "1.7.2",
-    "lockfileVersion": 2,
```

Comment:
Let me know what happened here, I just ran`npm install wait-port --save-dev` and a lot seemed to change

Reply:
Lockfiles can change wildly with updates.

Reply:
I think I might be using npm 6, whilst this `package-lockfile.json` was created with npm 7; I had npm 6 locally.

The key for me was that it actually was decreasing the lockfile version, which really did not look right to me.

I did:

```
npm install -g npm@latest

npm install
```

the diff should be far smaller

Reply:
Yes, npm 6 actually fails on the Dynamo.js file. We need npm 7.  Can't wait till the next node goes LTS with NPM 7.

Reply:
A candidate for a separate merge could be setting engines in the package.json to specify minimum node versions and npm versions:

```json
"engines": {
    "node": ">=12.0.0",
    "npm": ">=7.0.0"
  },
```
This would give a friendly error I think

Reply:
Yes please !!